.github/workflows/ci.yml

```diff
@@ -67,7 +82,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: 13
+          node-version: 10
```

Comment:
Why did you change this line? Each step is independent and can have different environment.

Reply:
Hi @TomaszMolenda,
I wanted to unify node-version that is aligned with node version that is used by Google Cloud Functions. GCP Cloud Functions are using Node 10.

To remove uncertainty if the feature is available in Node 10 or not (and by accident not use a feature from Node 13), it makes sense, to just use same version of NodeJS. 

Same things I did with `@types/node`.

If that doesn't make sense, let me know.

Reply:
I think it's a good idea to keep the same version.

Reply:
> Hi @TomaszMolenda,
> I wanted to unify node-version that is aligned with node version that is used by Google Cloud Functions. GCP Cloud Functions are using Node 10.
> 
> To remove uncertainty if the feature is available in Node 10 or not (and by accident not use a feature from Node 13), it makes sense, to just use same version of NodeJS.
> 
> Same things I did with `@types/node`.
> 
> If that doesn't make sense, let me know.

You're right. I think it's a good idea to keep compatibility version with GCP.